### PR TITLE
Solid frontpage integration

### PR DESCRIPTION
**Pull request by webblocksapp**
- Added SolidJS framework icon.
- Updated `README.md` with newer steps for running frontpage docs.
- Command `link-monorepo-docs` was deprecated because is not working. New steps for running the project were added at `README.md`
- SolidJS code snippets and the `sync.js` file were added at [monorepo PR 21207](https://github.com/storybookjs/storybook/pull/21207). Monorepo PR needs to be merged first.

### CODE REVIEW

File: README.md
Comment (winkerVSbecks) [inline README.md:52]: the `$branch` part is correct but the rest should stay as is. There two workflows: one for editing content from a symmlink, the other to pull content from the monorepo

**Comment by winkerVSbecks** [comment]
@webblocksapp thanks for the opening this PR. However, it doesn't include the `sync.js` file. 

@kylegach could you review this new workflow too please

**Comment by webblocksapp** [comment]
Hi @winkerVSbecks, the `sync.js` file lives in the monorepo.

**Comment by winkerVSbecks** [comment]
@webblocksapp could you link me to it. I don't see it on the `next` branch.

**Comment by webblocksapp** [comment]
@winkerVSbecks sorry, I forgot to mention here the related PR where `sync.js` was added. I've just updated the PR description with the monorepo PR link that adds the SolidJS snippets as well.

File: README.md
Comment (winkerVSbecks) [reply README.md:52]: Thanks for that additional context. The PR looks good to me. Do you want to see any additional changes before I merge it?

**Review by winkerVSbecks — APPROVED** [review]

File: README.md
Comment (kylegach) [reply README.md:52]: Ahh, I hadn't seen the `sync` script added to the monorepo. That's about 100x times better than my suggestion above. 🙌

File: README.md
Comment (kylegach) [inline README.md:77]: If you agree with my suggestion to ask for a relative path instead ([see comment on other PR](https://github.com/storybookjs/storybook/pull/21207#discussion_r1120319949)), then these instructions should be updated as well.

**Review by kylegach — APPROVED** [review]
I left one minor comment, but this is good-to-go! Thanks, @webblocksapp!

**Comment by kylegach** [comment]
We can ignore the CircleCI failure. It seems we don't have that configured for outside forks.

**Comment by jonniebigodes** [comment]
To avoid accidentally merging this, I've added the `Do not merge` label as we will hold off on getting the snippets added for the documentation until Storybook 7.0 is out and with that, enlist someone from the SolidJS community that's more knowledgeable of the framework to help us and @webblocksapp get the snippets in. 

@webblocksapp, will that work for you? I left a similar comment in the companion pr in the Storybook monorepo.

**Comment by webblocksapp** [comment]
Readme has been updated with the new "synch docs task".

**Pull request merged**
